beets/ui/__init__.py

```diff
@@ -357,6 +357,13 @@ def human_seconds_short(interval):
                 "fuchsia", "turquoise", "white"]
 RESET_COLOR = COLOR_ESCAPE + "39;49;00m"
 
+# Map the color names to the configured colors in a dict
+COLOR_NAMES = ['text_success', 'text_warning', 'text_error', 'text_highlight',
+               'text_highlight_minor', 'action_default', 'action']
+COLORS = dict(zip(COLOR_NAMES,
+                  map(lambda x: config['ui']['colors'][x].get(str),
+                      COLOR_NAMES)))
```

Comment:
Also, although I know it's inefficient, can we please move this lookup inside the `colorize` function? I would like to avoid config lookups on module import, if at all possible—they can be somewhat hard to predict and can cause slow startup time.